### PR TITLE
Adds a gas planet as a random parallax layer

### PR DESCRIPTION
**Pull request by ghost**
:cl: Qustinnus, Dorsidwarf
spriteadd: The station can now sometimes be orbiting a gas planet, be sure to look out of the window!
/:cl:

i hate the "ss13 orbits a gas planet" meme-lore, but this atleast makes it so that you're only /sometimes/ orbiting one and as a centrist I can live with that.

also killed off the prob(70) in the random layer selection because theres enough variation now and it can just always pick one.

image
image

### CODE REVIEW

**Comment by tralezab** [comment]
trips say this must get in, looks dope

File: code/controllers/subsystem/parallax.dm [hunk 1]
```diff
@@ -14,9 +14,8 @@ SUBSYSTEM_DEF(parallax)
 //These are cached per client so needs to be done asap so people joining at roundstart do not miss these.
 /datum/controller/subsystem/parallax/PreInit()
 	. = ..()
-	if(prob(70))	//70% chance to pick a special extra layer
-		random_layer = pick(/atom/movable/screen/parallax_layer/random/space_gas, /atom/movable/screen/parallax_layer/random/asteroids)
-		random_parallax_color = pick(COLOR_TEAL, COLOR_GREEN, COLOR_SILVER, COLOR_YELLOW, COLOR_CYAN, COLOR_ORANGE, COLOR_PURPLE)//Special color for random_layer1. Has to be done here so everyone sees the same color.
+	random_layer = /atom/movable/screen/parallax_layer/planet/gas, /atom/movable/screen/parallax_layer/random/space_gas, /atom/movable/screen/parallax_layer/random/asteroids)
```
Comment (LemonInTheDark) [inline code/controllers/subsystem/parallax.dm]: bruh

File: code/controllers/subsystem/parallax.dm [hunk 2]
```diff
@@ -14,9 +14,8 @@ SUBSYSTEM_DEF(parallax)
 //These are cached per client so needs to be done asap so people joining at roundstart do not miss these.
 /datum/controller/subsystem/parallax/PreInit()
 	. = ..()
-	if(prob(70))	//70% chance to pick a special extra layer
```
Comment (LemonInTheDark) [inline code/controllers/subsystem/parallax.dm:17]: you killing off this behavior?

Comment (Qustinnus) [reply code/controllers/subsystem/parallax.dm:17]: ah yeah sorry i will mention it

File: code/controllers/subsystem/parallax.dm (hunk 1 above)
Comment (Qustinnus) [reply code/controllers/subsystem/parallax.dm]: moment

**Comment by CRITAWAKETS** [comment]
that looks terrible because of different pixel resolutions between the planet and the gas giant which you can clearly see mostly because they're on top of eachother.

also, the SS13 orbits a gas giant lore is from before lavaland

**Comment by Qustinnus** [comment]
i dont care if it was before lavaland it was dumb in the first place and was used for countless weird PRs when the gas planet thing was mentioned off-hand in a wiki-page like once.

**Comment by Qustinnus** [comment]
also how small do you think gas giants are?

**Comment by CRITAWAKETS** [comment]
> also how small do you think gas giants are?

how light do you think gravity is

**Comment by Qustinnus** [comment]
image
I repeat do you not realize how large they are???

**Comment by Qustinnus** [comment]
gas giants are not known for being very dense (hint: its gas), they can thus be humongous without gaining as much gravitational pull for the same growth in size as a solid planet would.

**Comment by Qustinnus** [comment]
image
a final representation for if you are still bewildered by the fact that gas giants, are in fact, very big. Titan is in the bottom left corner.

**Comment by Sparkezel** [comment]
What honestly wonders me is how big does a gas giant look like from one of thier moons/station's orbit

**Comment by interestingusernam3** [comment]
Is this a plasma planet or a fart gas planet

**Comment by Iamgoofball** [comment]
gas giants are huge yes but this is too big

**Comment by Qustinnus** [comment]
did you not see the last image I posted goof?

**Comment by Jared-Fogle** [comment]
i love pictures of saturn

hey @EOBGames is this lore thematic

**Comment by Iamgoofball** [comment]
@Qustinnus realism is not an excuse, make it just a bit smaller please

**Comment by LemonInTheDark** [comment]
I kinda agree with goof, realism is less important then how the art looks in comparison with the rest of the game. If it doesn't look good, why bother?

Now I'm not known for my artistic eye, we have twat for that, but arguing around realism is useless here

**Review by Twaticus — CHANGES_REQUESTED** [review]
its so drastically different from everything else it stands out like a sore thumb. and i like the idea of it being giant but realistically it takes up so much room that every window to space is just going to be purple. smaller and matching please

**Comment by Qustinnus** [comment]
alright closing 4 now then

**Pull request closed without merging**